FinSearchUnified/Components/ConfigLoader.php

```diff
@@ -97,7 +98,7 @@ private function getConfigFile()
      */
     private function getCacheKey()
     {
-        return sprintf('%s_%s', self::CACHE_ID, $this->shopkey);
+        return StaticHelper::removeSpecialCharacters(sprintf('%s_%s', self::CACHE_ID, $this->shopkey));
```

Comment:
Was there a specific reason to use `StaticHelper::removeSpecialCharacters`? The only case where you could get an invalid cache key is when you enter a wrong shopkey, which is unlikely to happen (assuming the customer did nothing wrong).

Reply:
Yes apparently we are testing for an invalid shopkey in the test which has spaces so it was failing for the "Findologic Shopkey"

Reply:
Which test is that exactly? I just ran all of them using `return sprintf('%s_%s', self::CACHE_ID, $this->shopkey);` and they passed.